app/jinja_filters.py

```diff
@@ -463,6 +463,7 @@ def __init__(  # noqa: C901, R0912 pylint: disable=too-complex, too-many-branche
                 multiple_answers
                 or answer_type == "relationship"
                 or summary_type == "CalculatedSummary"
+                or summary_type == "GrandCalculatedSummary"
```

Comment:
Minor and non-blocking, could use a constant or Enum given we have got the literal in a lot.

Reply:
gone with a function instead since the summary type check is always against both - arguably I could have just made the type of the block a CalculatedSummary for both but its probably clearer to keep it separate